tdrs-frontend/cypress/e2e/common-steps/data_files.js

```diff
@@ -10,15 +10,15 @@ export const restartAtHomePage = () => {
 }
 
 export const fillFYQ = (fiscal_year, quarter) => {
+  cy.wait(1000)
   cy.get('#reportingYears').should('exist').select(fiscal_year)
```

Comment:
in general, i prefer expanding the timeout on an assertion step so the test doesn't wait longer than it has to. doesn't work in all scenarios though

```suggestion
  cy.get('#reportingYears', { timeout: 10000 }).should('exist').select(fiscal_year)
```